lib/server.js

```diff
@@ -124,7 +124,8 @@ function create(log, error, config, routes, db, translator) {
     },
     load: {
       sampleInterval: 1000
-    }
+    },
+    useDomains: false
```

Comment:
This made it so uncaught exceptions in our handlers and in the pre and post events would be caught in node's `Domain`, and then hapi would just output with it's quirky `server.log()` mechanism. This meant we didn't know about some errors. Domains are weird, and considered a mistake in nodejs, so lets just disable it, so we never hide exceptions.